include/rgl/api/core.h

```diff
@@ -308,6 +328,18 @@ rgl_entity_destroy(rgl_entity_t entity);
 RGL_API rgl_status_t
 rgl_entity_set_pose(rgl_entity_t entity, const rgl_mat3x4f *local_to_world_tf);
 
+/******************************** SCENE ********************************/
+
+/**
+ * Sets time to the given scene.
+ * Time indicates a specific point in time, relative to a clock's 0 point.
+ * Timestamp is used to fill field RGL_FIELD_TIME_STAMP_F64 or for ROS2 publishing.
+ * @param scene Scene where time will be set. Pass NULL to use the default scene.
+ * @param nanoseconds Timestamp in nanoseconds.
+ */
+RGL_API rgl_status_t
+rgl_scene_set_time(rgl_scene_t scene, uint64_t nanoseconds);
```

Comment:
I wonder if it is the right interface. Although it will work for the next 500 years, I doubt it is natural to express time in nanoseconds. Could you have a look what interface would be most convenient for currently existing plugins and share thoughts?

Reply:
I have modeled it on the ROS2 time format: two ints32 for seconds and nanoseconds. I have simplified that into one field. My wonder was to use `double seconds` or `uint64_t nanoseconds`. The first option was more natural, but no resolution was guaranteed (milliseconds or nanoseconds?). Since this time is used to work with ROS2, I wanted to ensure that the time set will be exactly the same as provided.